client/src/lib/components/EmailCreator/EmailCreator.svelte

```diff
@@ -0,0 +1,84 @@
+<script lang="ts">
+  export let contactEmails: string[];
```

Comment:
En gros ici j'ai sorti la partie du formulaire qui permettait de créer des emails de contact dans un autre composant.

@DaFrenchFrog indiquait [ici](https://lite.framacalc.org/0c5a3v0z3t-9th0) que dans la première itération il n'y aurait qu'une adresse email de contact.

Comme je trouvais ce composant pas mal foutu, je le garde au chaud dans un fichier pour le réutiliser plus tard

Reply:
On dirait qu'on va pouvoir le réutiliser tout de suite :-)

Par contre je l'appellerais peut-être plus explicitement `ContactEmailsField` ? "EmailCreator" ça pourrait aussi être une sorte de générateur d'email par ex...